packages/hooks/use-draggable/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import {useEffect} from "react";
+
+export interface UseDraggableProps {
+  /**
+   * Ref to the moving target DOM node.
+   */
+  targetRef?: React.RefObject<HTMLElement>;
+  /**
+   * Ref to the drag DOM node.
+   */
+  dragRef?: React.RefObject<HTMLElement>;
+  /**
+   * Whether the target is draggable.
+   * @default true
+   */
+  draggable?: boolean;
+  /**
+   * Whether the target can overflow the viewport.
+   */
+  overflow?: boolean;
+}
+
+export function useDraggable(props: UseDraggableProps) {
```

Comment:
We are using react-aria in our project. For consistency, could you consider using [useDrag hooks](https://react-spectrum.adobe.com/react-aria/useDrag.html)? 
Maintaining our own hook for drag operations is costly.

Reply:
> We are using react-aria in our project. For consistency, could you consider using [useDrag hooks](https://react-spectrum.adobe.com/react-aria/useDrag.html)? Maintaining our own hook for drag operations is costly.

I looked at useDrag and it shouldn't seem to be possible. 
IMO, useDrag is used to move some data to a location, while useDraggable is just to move the location of a Dom Element.

Reply:
Sorry I suggested the wrong hook earlier. How about considering [useMove](https://react-spectrum.adobe.com/react-aria/useMove.html)? 
It's cross-device compatible, which could be ideal for our needs.

Reply:
@ryo-manba @wingkwong Sorry, It took so long to reply, my time is limited on weekdays. I have refactored useDraggable base on useMove, cr pls.